### DISCUSSION TITLE
Circular Waveguide Simulation in Python

### DISCUSSION THREAD

**Discussion by ssmulla** [thliebig/openEMS-Project#76] [General]
How to define ports for circular waveguide in python? I am trying to use the "openEMS.ports.WaveguidePort" class, but I have the problem in defining E_WG_func, H_WG_func. Please help.

**Comment by thliebig** [comment]
I thinks this is not yet implemented for python. but it is in Octave/Matlab. maybe you can have a look there and try to implement/port it to python? That would be nice...

**Reply by ssmulla** [reply]
I will try it for circular waveguide,

At this time, I have designed and implemented a dipole antenna and a circular patch antenna using python, can you include it in the python examples?
